### PR TITLE
Feat subcommand

### PR DESCRIPTION
**Pull request by George-Miao**
This PR merge two binaries into one while keep them as subcommand:
```bash
suimu check [options]
```
Or
 ```bash
suimu build [options]
```

### CODE REVIEW

**Comment by outloudvi** [comment]
Hi! Please also update the executable names in `.github/workflows/release.yml`. Thanks!

**Comment by George-Miao** [comment]
> Hi! Please also update the executable names in `.github/workflows/release.yml`. Thanks!

Done

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -28,13 +28,12 @@ jobs:
         run: |
           pushd target/release
           rm suimu-*.d
-          mv suimu-check suimu-check-${{ runner.os }}
-          mv suimu-build suimu-build-${{ runner.os }}
+          mv suimu-check suimu-${{ runner.os }}
```
Comment (outloudvi) [inline .github/workflows/release.yml]: Maybe we shall change `suimu-check` to `suimu`?

Comment (George-Miao) [reply .github/workflows/release.yml]: Ah yes

**Pull request merged**